aliskin/assignment2/lca.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+
+class TreeNode():
```

Comment:
Could you factor out the common class TreeNode and build_tree_from_preorder() in another file so you don't have to duplicate the code?